packages/hierarchies-react/src/test/internal/TreeActions.test.ts

```diff
@@ -525,57 +579,61 @@ describe("TreeActions", () => {
 
       const actions = createActions(model);
 
-      actions.setInstanceFilter("root-1", filter);
-
-      expect(onModelChangedStub).to.be.calledOnce;
+      await actions.setInstanceFilter("root-1", filter)?.complete;
+      await waitFor(() => expect(onModelChangedStub).to.be.calledTwice);
       const newModel = onModelChangedStub.firstCall.args[0];
       expect(getHierarchyNode(newModel, "root-1")?.instanceFilter).to.be.eq(filter);
       expect(getHierarchyNode(newModel, "child-1")).to.be.undefined;
+      await waitFor(() => {
+        expect(provider.getNodes).to.be.calledWith(createGetNodesProps({ parentNode: getHierarchyNode(newModel, "root-1")?.nodeData, instanceFilter: filter }));
+      });
```

Comment:
I don't think `getNodes` check is needed for this test

Reply:
I think this was fixed